data/data/bootstrap/bootstrap-in-place/files/opt/openshift/bootstrap-in-place/bootstrap-in-place-post-reboot.sh

```diff
@@ -12,6 +12,18 @@ function wait_for_api {
   done
 }
 
+# This is required since the progress service (https://github.com/openshift/installer/blob/dd9047c4c119e942331f702a4b7da85c60042da5/data/data/bootstrap/files/usr/local/bin/report-progress.sh#L22-L33),
+# usually dedicated to creating the bootstrap ConfigMap, will fail to create this configmap in case of bootstrap-in-place single node deployment, 
+# due to the lack of a control plane when bootkube is complete
+function signal_bootstrap_complete {
+  until oc get cm bootstrap -n kube-system &> /dev/null
+  do
+    echo "Creating bootstrap configmap ..."
```

Comment:
Should this be printed at the top of the function rather than every 5 seconds when we are waiting?

Reply:
Yeah that's probably the right thing to do. However, I don't expect us to ever hit this echo more than once (at-least not under usual circumstances), because at this point the API server will most likely be up